src/zodbupdate/convert.py

```diff
@@ -13,42 +14,44 @@ class Datetime(datetime.datetime):
     def __reduce__(self):
         type_info, args = super(Datetime, self).__reduce__()
         assert len(args) > 0
-        return (datetime.datetime, (zodbpickle.binary(args[0]),) + args[1:])
```

Comment:
I believe this will do an extra ``encode('utf-8')`` here, and my gut feeling is that you really do not want to do this.